nabboot/nabboot.service

```diff
@@ -1,10 +1,12 @@
 [Unit]
 Description=Nabaztag boot script, initializing leds and motors
+Before=nginx.service
```

Comment:
Besides, nabboot.service should not depend on nginx. Indeed, this service is only there to setup leds and is totally unrelated to a web interface.

Reply:
As requested, I removed this.
But we are a bit playing on words here: indeed the `nabboot module` itself currently only deals with LEDs, but nothing prevents the `nabboot service `to deal with other boot related chores...
Maybe we should introduce a` nginx-hack service` to do this job? This would allow reintroducing the -p option of PiShrink....